src/patternfly/components/Drawer/drawer.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-drawer {
+  --pf-c-drawer__main--MarginRight: calc(var(--pf-c-drawer__panel--Width) * -1);
+  --pf-c-drawer--m-open__main--MarginRight: 0;
+  --pf-c-drawer__main--Transition: margin .25s ease;
+  --pf-c-drawer__panel--ZIndex: var(--pf-global--ZIndex--sm);
+  --pf-c-drawer__panel--Width: 100%;
+  --pf-c-drawer__panel--md--Width: #{pf-size-prem(450px)};
+  --pf-c-drawer__panel--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-drawer__panel--Transition: transform .25s ease;
+  --pf-c-drawer__panel--Transform: translateX(100%);
+  --pf-c-drawer--m-open__panel--Transform: translateX(0);
+  --pf-c-drawer--m-open__panel--BoxShadow: var(--pf-global--BoxShadow--md-left);
+  --pf-c-drawer__panel-body--PaddingTop: var(--pf-global--spacer--md);
```

Comment:
At this size, this is happening

<img width="641" alt="Screen Shot 2019-07-17 at 2 49 52 PM" src="https://user-images.githubusercontent.com/20118816/61402941-33477480-a8a2-11e9-8e54-35ba86355136.png">